### ISSUE TITLE
Zoomed image displayed to the side of original image

### ISSUE THREAD

**Issue by beckybair** [Intera/vue-zoom-on-hover#9]
I was just wondering if it is possible to have the zoomed image displayed to side of the original image instead of on top of - also have the zoomed in area grayed a bit on the original so you can see what part is zoomed?  Like Amazon does?  I hope this makes sense...

**Comment by jkalbhenn** [comment]
i dont think this is currently possible because the larger image and the smaller image are inside the same container. but maybe the `template:` of the component can be modified to make it possible. but i dont have a solution at the moment. it would be an interesting new feature.

**Comment by TnineInfotech** [comment]
Is it still open? It will be great if this was implemented. A lot of e-commerce use cases can be solved with this enhancement.

**Comment by jkalbhenn** [comment]
this is still open. not sure if i can implement it soon but im open to patches.
